property/index.js

```diff
@@ -93,6 +96,7 @@ module.exports = yeoman.generators.Base.extend({
         name: 'itemType',
         message: 'The type of array items:',
         type: 'list',
+        default: this.propDefinition && this.propDefinition.itemType,
```

Comment:
There is no `propDefinition.itemType` AFAICT, see also https://github.com/strongloop/generator-loopback/pull/200/files#r68249058